upgrade_guide/README.md

```diff
@@ -3,6 +3,19 @@
 The purpose of this guide is to detail changes made by successive
 versions of the Java driver.
 
+### 3.2.0
+
+[JAVA-1257](https://datastax-oss.atlassian.net/browse/JAVA-1257) has deprecated the `QueryLogger`
+class and users are now encouraged to use `EnhancedQueryLogger` instead, which makes it easier for
+users to customize the way statements are logged. 
+Please note that `QueryLogger` class might be removed in a future major release.
+
+[JAVA-1257](https://datastax-oss.atlassian.net/browse/JAVA-1257) also modifies the behavior 
```

Comment:
I don't think we ever documented the fact that it returned the Query string, but I'm glad you documented this here because it does sound like this could break somebody (even our tests assumed it worked this way).

The other possible concern is that `toString` will now be more intensive since it includes the parameter values and such.  If you have a ton of parameters that need to be serialized, repeated calls to `toString` could create some work - but I think that would be unusual.